mempool/src/shared_mempool/peer_manager.rs

```diff
@@ -31,7 +33,7 @@ use vm_validator::vm_validator::TransactionValidation;
 const PRIMARY_NETWORK_PREFERENCE: i64 = 0;
 
 /// Peers that receive txns from this node.
-pub(crate) type PeerInfo = HashMap<PeerNetworkId, PeerSyncState>;
+pub(crate) type PeerSyncStates = HashMap<PeerNetworkId, PeerSyncState>;
```

Comment:
the simple things...

Reply:
There's another object named `PeerInfo` in state sync, that's essentially the `PeerStateSync` here, so I needed it to be less confusing...

Reply:
haha, well a collection of PeerSyncState called PeerInfo is ...